src/main/java/com/vinylteam/vinyl/service/impl/DefaultDiscogsService.java

```diff
@@ -101,7 +101,14 @@ public String getDiscogsLink(String artist, String release, String fullName) thr
         return discogsLink;
     }
 
+    public DiscogsClient getDiscogsClient() {
+        return discogsClient;
```

Comment:
why do you return client? It shouldn't be returned anywhere

Reply:
Changed. I added one test for testing wantlist() method from DiscogsClient. So method getDiscogsClient() was deleted